src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj

```diff
@@ -0,0 +1,49 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+  <PropertyGroup>
+    <TargetFramework>$(LatestTargetFramework)</TargetFramework>
+    <!-- Workaround https://github.com/microsoft/MSBuildSdks/issues/469 -->
+    <TargetFrameworks />
+
+    <Stage>transport</Stage>
+    <IsShipping>false</IsShipping>
+
+    <PackageDescription>Internal transport package to provide dotnet-api-docs with the reference assemblies and compiler generated documentation files from dotnet/extensions.</PackageDescription>
+    <IncludeBuildOutput>true</IncludeBuildOutput>
+    <IncludeSymbols>false</IncludeSymbols>
+    <EnablePackageValidation>false</EnablePackageValidation>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <NoWarn>$(NoWarn);NU5128;NU5131</NoWarn>
+    <BuildProjectReferences Condition="'$(NoBuild)' == 'true'">false</BuildProjectReferences>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);IncludeProjectReferenceCompilerGeneratedSecondaryOutputInPackage</TargetsForTfmSpecificContentInPackage>
+    <!-- Reference the outputs to have them available as build outputs. -->
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Set PrivateAssets to avoid dependencies in the nupkg
+         Set Private=false to avoid copying the referenced projects to output -->
+    <ProjectReference PrivateAssets="all"
+      Private="false"
+      Include="$(SrcLibrariesDir)\*\*.*proj;
+               $(SrcLibrariesDir)\..\ToBeMoved\*\*.*proj;
+               $(SrcLibrariesDir)\..\ToBeRemoved\*\*.*proj"
```

Comment:
[nit] this should have been removed

Reply:
Addressed in https://github.com/dotnet/extensions/pull/4562